sweep/store.go

```diff
@@ -122,6 +122,13 @@ type SweeperStore interface {
 
 	// ListSweeps lists all the sweeps we have successfully published.
 	ListSweeps() ([]chainhash.Hash, error)
+
+	// GetTx queries the database to find the tx that matches the given
+	// txid. Returns ErrTxNotFound if it cannot be found.
+	GetTx(hash chainhash.Hash) (*TxRecord, error)
+
+	// DeleteTx removes a tx specified by the hash from the store.
+	DeleteTx(hash chainhash.Hash) error
```

Comment:
👀 ohh we never garbage  collected before, but we aren't going to use it in this PR ?

Reply:
yeah it's used in a later PR I think.